File: sync/v1/sync_list_item/create-create-item/create-create-item.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Collections.Generic;
using Twilio;
using Twilio.Rest.Sync.V1.Service.SyncList;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var data = new Dictionary<string, Object>()
        {
            {"number", "001"},
            {"name", "Bulbasaur"},
            {"attack", 49}
        };

        var syncListItem = SyncListItemResource.Create(
            ttl: 864000,
            data: data,
            pathServiceSid: "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
            pathListSid: "MyCollection"
        );

        Console.WriteLine(syncListItem.Index);
    }
}

File: sync/v1/sync_list_item/create-create-item/create-create-item.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

sync_list_item = @client.sync
                        .services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
                        .sync_lists('MyCollection')
                        .sync_list_items
                        .create(ttl: 864000, data: {
                             'number' => '001',
                             'name' => 'Bulbasaur',
                             'attack' => 49
                         })

puts sync_list_item.index
